tools/deprecate_version/deprecate_version.py

```diff
@@ -24,7 +24,7 @@
 import re
 import sys
 
-import github
+import github  # pip install pygithub
```

Comment:
If you run https://github.com/envoyproxy/envoy/blob/master/tools/deprecate_version/deprecate_version.sh everything should run in a venv and work correctly?